src/include/sof/schedule/dp_schedule.h

```diff
@@ -67,13 +67,9 @@
  */
 int scheduler_dp_init(void);
 
-/**
- * \brief Set the Data Processing scheduler to be accessible at secondary cores
- */
-int scheduler_dp_init_secondary_core(void);
-
 /**
  * \brief initialize a DP task and add it to scheduling
+ * This may be called at any core, not necessarily on the one the task will be scheduled on
```

Comment:
seems have some grammar mistakes?
"not necessarily on the one the task will be scheduled on"

Reply:
Even worse, comments have not been changes and are not true anymore
Now the threat must be created on the same core it will run on. 
Comments changed, assert added